atd-etl/afd_ems_import/afd_incident_upload.py

```diff
@@ -0,0 +1,384 @@
+#!/usr/bin/env python
+
+"""
+Name: AFD Incident Uploads
+Description: This flow uploads AFD Incident Response CSVs
+    The data is emailed to atd-afd-incident-data@austinmobility.io daily ~ 6:30AM. From there it
+    gets forwarded to a S3 bucket via AWS Simple Email Service.
+"""
+
+import datetime
+import json
+import boto3
+import pprint
+import os
+import time
+import email
+import tempfile
+import shutil
+import pandas
+from datetime import datetime, timedelta
+import psycopg2
+import psycopg2.extras
+from sshtunnel import SSHTunnelForwarder
+
+import onepasswordconnectsdk
+from onepasswordconnectsdk.client import Client, new_client
+
+pp = pprint.PrettyPrinter(indent=4)
+
+DEPLOYMENT_ENVIRONMENT = os.environ.get(
+    "ENVIRONMENT", "development"
+)  # our current environment from ['production', 'development']
+ONEPASSWORD_CONNECT_TOKEN = os.getenv("OP_API_TOKEN")  # our secret to get secrets 🤐
+ONEPASSWORD_CONNECT_HOST = os.getenv("OP_CONNECT")  # where we get our secrets
+VAULT_ID = os.getenv("OP_VAULT_ID")
+
+
+def main():
+    secrets = get_secrets()
+
+    global DB_USERNAME
```

Comment:
I know that I [left you hanging](https://github.com/cityofaustin/atd-vz-data/pull/1235/files#r1245923996) that last time I brought this up, but I'm starting to think that this ties into the 1PW in the DAG vs 1PW in the container discussion. I'm starting to feel like the Docker containers should not be concerned with how the secrets are obtained but only with receiving them. It would help avoid a refactor if we ever shifted from the 1PW server (which is unlikely and I surely hope we never have to!).

Like you mentioned today, it is a lot to think about, and I'll keep thinking about it over the weekend.